defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+sansible_prom2teams:
+  checksum: sha256:407e0311689207b385fb1252f36d3c3119ae9a315e3eba205aaa69d576434ed7
```

Comment:
Can you alphabetically order these vars?

Reply:
Done.